src/widgets/settingspages/ExternalToolsPage.hpp

```diff
@@ -14,8 +14,10 @@ class ExternalToolsPage : public SettingsPage
 
 private:
     void initLayout(GeneralPageView &layout);
+    void exportSettings();
+    void importSettings();
```

Comment:
I'd prefer if these functions were source-only, and accepted a `QWidget *parent` parameter instead of using `this` for the dialog parents. You can still pass `this` to them. They should also be renamed to clarify that they are specifically importing or exporting image uploader settings, not any generic settings.